packages/slice-machine/src/legacy/components/Navigation/index.tsx

```diff
@@ -25,10 +24,6 @@ import { FolderIcon } from "@/icons/FolderIcon";
 import { LightningIcon } from "@/icons/Lightning";
 import { MasterSliceLibraryIcon } from "@/icons/MasterSliceLibraryIcon";
 import { SettingsIcon } from "@/icons/SettingsIcon";
-import VideoItem from "@/legacy/components/Navigation/VideoItem";
```

Comment:
Can do that, I didn't do it right away because I see that that it was quite a few lines of code, and we could want to bring it back in the future 🤷‍♂️

Reply:
Same:
> Prevent redoing the same comment there but yeah If product wants us to remove the menu item totally, let's remove ALL code related to that, components, tests, everything


I guess it depends on what Côme said, if they want to remove it totally and don't know about a usage again, let's remove it totally. We can find it after anyway with git history